### PR TITLE
Feat/FRO-10:implenmented upload of files on zurichat

### PR DESCRIPTION
**Pull request by Princessayomide53**
**Fixes Issue/Linear Ticket-FRO-10**

FRO-10-SELECTING OF MULTIPLE FILES  

https://linear.app/team-socket/issue/FRO-10/selecting-of-multiple-files

My PR Fixes ID-FRO-10

**Changes proposed**

**What were you told to do?**

I was told to fix the selecting of files and images

**What did you do?**

•upgraded my node version to 17.0.0 
It was initially version 14.5.0 
Without the installation I could not run the files on my local system

•I worked on the uploads of files
Previously zurichat messaging can only select a image or files and when they do select another image or files it disappears 

•I made sure it was possible to select multiple files and images in the input box

Check List (Check all the applicable boxes)

🚨**Please review the [style guide for contributing]**(https://github.com/zurichat/zc_main/blob/dev/docs/STYLING.md) and [guidelines for contributing](https://github.com/zurichat/zc_main/blob/dev/docs/CONTRIBUTING.md) to this repository.

 ✔️ My code follows the code style of    this project.
 ✔️ This PR does not contain plagiarized content.
 ✔️ The title and description of the PR is clear and explains the approach.
✔️  I am making a pull request against the dev branch (left side).
 ✔️ My commit messages styles matches our requested structure.
 ✔️ My code additions will fail neither code linting checks nor unit test.
 ✔️ I am only making changes to files I was requested to.

### CODE REVIEW

**Comment by Princessayomide53** [comment]
03E49BDC-03E1-47D1-9CB8-F63A0D124551


Screenshot of selecting multiple uploads

**Comment by Princessayomide53** [comment]
Uploading IMG_7892.MOV…



selecting multiple files uploads

**Comment by Princessayomide53** [comment]
selecting of multiple files upload

**Comment by Princessayomide53** [comment]
selecting of files upload

File: package.json [hunk 1]
```diff
@@ -49,5 +49,8 @@
     "pretty-quick": "^3.1.1",
     "rimraf": "^3.0.2",
     "serve": "^12.0.0"
+  },
+  "dependencies": {
+    "node": "^17.0.0"
```
Comment (ickynavigator) [inline package.json:54]: why did you add node as a dependency?

**Review by ickynavigator — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Princessayomide53) [reply package.json]: I updated my node js and I guess the package.json got affected 
I made changes already 
Thank you
